dashboard/db/migrate/20170822201210_backfill_authorized_teacher_user_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class BackfillAuthorizedTeacherUserPermissions < ActiveRecord::Migration[5.0]
+  def up
+    Plc::UserCourseEnrollment.find_each do |enrollment|
+      UserPermission.create(permission: UserPermission::AUTHORIZED_TEACHER, user_id: enrollment.user_id)
```

Comment:
Does it make sense to use [`find_or_create_by`](https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by) to convey that there is no need to create a new `UserPermission` if one already exists? Also below.